packages/design-system/src/components/N8nCallout/Callout.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+	<div :class="classes" role="alert">
+		<div :class="this.$style.icon">
```

Comment:
you don't need `this.$style`. can be just `$style`

Reply:
Ok, changed.